packages/popper/src/react-popper.ts

```diff
@@ -69,13 +72,16 @@ export function usePopper(
       enabled: true,
```

Comment:
How about passing the `enabled` option here? instead of adding the `if` block below

Reply:
Ha! I didn't realized that's an option. Great suggestion, thanks!

I updated the PR.